source/_integrations/rtsp_to_webrtc.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: RTSPtoWebRTC
+description: Instructions on how to integrate RTSPtoWebRTC server within Home Assistant.
+ha_category:
+  - Other
```

Comment:
Question: Does it make sense to include `Camera` here since its a modification to the camera integration? It does not provide camera entities itself, so it wasn't clear to me.

Reply:
Maybe? Dunno... Not having it there is maybe equally confusing?

Reply:
OK, i replaced other with camera.